bpf/run_probes.sh

```diff
@@ -46,11 +46,12 @@ function probe_kernel_config()
     # Other distros in /boot/config-*
```

Comment:
If you change the commit message to `Fixes: #0000` it will automatically close the issue when we merge.

Reply:
I can fix this up when I squash the patches back.